src/Build.UnitTests/Instance/ProjectInstance_Internal_Tests.cs

```diff
@@ -1,22 +1,18 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System.Collections.Generic;
-using Microsoft.Build.Execution;
-using Microsoft.Build.Evaluation;
-using Microsoft.Build.Framework;
-using System.Collections;
 using System;
-using System.Diagnostics;
-using Microsoft.Build.Construction;
+using System.Collections.Generic;
 using System.IO;
-using System.Xml;
 using System.Linq;
+using System.Xml;
 using Microsoft.Build.BackEnd;
-using Microsoft.Build.Engine.UnitTests;
+using Microsoft.Build.Construction;
+using Microsoft.Build.Evaluation;
+using Microsoft.Build.Execution;
+using Microsoft.Build.Framework;
```

Comment:
Add an AddTarget test to assert that it picks up the new target lists?

Reply:
`AddTarget` currently has no UTs, but sure I can add one to help improve coverage.